### ISSUE TITLE
I am probably an idiot

### ISSUE THREAD

**Issue by willson556** [GetiPlayerAutomator/get-iplayer-automator#278]
_From [tspill...@dcmtm.com](https://code.google.com/u/106857344145723919627/) on October 03, 2013 05:52:06_

here is my question...and. like the  subject line says..i am probably an idiot.big fan of Will Rogers - "We are all ignorant about something"
Here is my ?
why when doing downloads does the file size change drastically? I get a fluctuation here and there by tiny bits
I have often seen it go from 594 mb (example) to 1200 MB 
or sometimes sit there and hover a few hundred MB apart and change form small to large in front of my eyes
in other words will start saying the file size is 750 then go up to 1100 while downloading and fluctuate up and down
Like I say - ignorant am i..but isnt a file size a file size?

_Original issue: http://code.google.com/p/get-iplayer-automator/issues/detail?id=286_
